data/test/vtgate/aggr_cases.txt

```diff
@@ -272,6 +272,51 @@
   }
 }
 
+# group by a unique vindex should revert to simple route, even if aggr is complex
+"select id, 1+count(*) from user group by id"
+{
+  "Original": "select id, 1+count(*) from user group by id",
+  "Instructions": {
+    "Opcode": "SelectScatter",
+    "Keyspace": {
+      "Name": "user",
+      "Sharded": true
+    },
+    "Query": "select id, 1 + count(*) from user group by id",
+    "FieldQuery": "select id, 1 + count(*) from user where 1 != 1 group by id"
+  }
+}
+
+# group by a unique vindex where expression is qualified (alias should be ignored)
```

Comment:
Also I would like to see a test case showing that this is supported where the unique vindex is aliased, e.g. `select id as x, 1+count(*) from user group by x`

Reply:
Also (again for completeness) it would be good to add a test case that this is supported when grouping by a unique field AND additional fields.